test/parameter_test.py

```diff
@@ -261,6 +275,16 @@ def test_enum_param_invalid(self):
     def test_enum_param_missing(self):
         self.assertRaises(ParameterException, lambda: luigi.parameter.EnumParameter())
 
+    def test_list_parameter(self):
```

Comment:
I'll work on it

I have written config tests. But am struggling with the cli tests. I'll be redoing these commits and they will be forthcoming.